src/suite.py

```diff
@@ -533,36 +450,78 @@ def all_combinations(self, function, matched_pairs=False):
 
   # training the model
   def self_issue_classification_all(self, matched_pairs=False):
-
-    def self_issue_classification_statistics_per():
-      # self.train_data
-      self.self_issue_classification_from_comments()
-      score = fbeta_score(
-          self.train_data["prediction"].tolist(),
-          self.train_data["label"].tolist(),
-          average="weighted",
-          beta=0.5)
-
-      return score
-
-    best_score = self.all_combinations(
-        self_issue_classification_statistics_per, matched_pairs=matched_pairs)
-    logging.info("Best f0.5 score: {}".format(best_score))
-    return self.model
+    global model

```

Comment:
global variable? are you sure you need it?

Reply:
It's always there.. You are right.. It's causing the confusion I discussed with you yesterday, about whether the model is updating itself with more data. I'll remove it.. in a new PR =p

> This is a big PR so it's hard to evaluate comprehensively. Putting your 6 changes into different PRs would make your reviewer's life easier. :)

Right.. adding features and changing data size can certainly be a separate PR.